pkg/server/metrics/metrics.go

```diff
@@ -70,10 +71,19 @@ func newServerMetrics() *ServerMetrics {
 			"service_method",
 		},
 	)
+	backend := prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: namespace,
+			Subsystem: subsystem,
+			Name:      "Number of konnectivity agent connected to the proxy server",
+		},
+		[]string{},
+	)
 	
 	prometheus.MustRegister(latencies)
 	prometheus.MustRegister(connections)
-	return &ServerMetrics{latencies: latencies, connections: connections}
+	prometheus.MustRegister(backend)
+	return &ServerMetrics{latencies: latencies, connections: connections, backend: backend}
```

Comment:
Please break the fields one per line with every one with a comma. This will help minimize future merge conflicts.